File: benchmarks/node/map.js
const counts = new Map();
for (let i = 0; i < 300000; i++) counts.set("key" + i, i);
let sum = 0;
for (let i = 0; i < 300000; i++) sum += counts.get("key" + i);
console.log(counts.size, sum);

File: benchmarks/go/map.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	counts := map[string]int64{}
	for i := int64(0); i < 300000; i++ {
		counts["key"+strconv.FormatInt(i, 10)] = i
	}
	var sum int64
	for i := int64(0); i < 300000; i++ {
		sum += counts["key"+strconv.FormatInt(i, 10)]
	}
	fmt.Println(len(counts), sum)
}
